woocommerce/woocommerce-square
# Display ApplePay on block cart and block checkout.

### All Submissions:



---

### Changes proposed in this Pull Request:


Fixes an issue in which the Apple Pay button was not displaying on the block cart and checkout pages for verified domains.

Closes #113

### Steps to test the changes in this Pull Request:

1. Install and configure the plugin on a publicly accessible site (ngrok works for local servers)
2. Configure the site to use block cart and checkout 
3. Log in to the Square dashboard
4. Download an add the domain verification file to the `/.well-known` directory of the site
5. Add the site's domain to the Apple Pay
6. Configure Square in the dashboard of WooCommerce
7. Enable Apple Pay/digital wallets
8. Create a simple product
9. In Safari go to the products page, ensure the Apple Pay button displays and allows you to purchase the product
10. Add the product to the cart
11. Visit the cart page, ensure the Apple Pay button displays and allows you to purchase the product
12. Add the product to the cart
13. Visit the checkout page, ensure the Apple Pay button displays and allows you to purchase the product
14. Configure the site to use shortcode cart and checkout
15. Add the product to the cart
16. Visit the cart page, ensure the Apple Pay button displays and allows you to purchase the product
17. Add the product to the cart
18. Visit the checkout page, ensure the Apple Pay button displays and allows you to purchase the product

Note: Geographical limitation prevent the use of Apple Pay in some countries.

### Changelog entry


> Fix - Apple Pay button is now available on Cart and Checkout Block pages.
